website/utilitypage_mine.py

```diff
@@ -102,6 +122,21 @@ def geocode (addressfull):  #geocoder address to coordinates
         return "Address Unclear", "Consider Rewriting"
 
 
+hostname = 'rapid-1304.vm.duke.edu'
+port = '5432'
+username = 'group3_read'
+password = 'water3all4me'
```

Comment:
perhaps we ought to read these from environmental variables. you could set an environmental variable on the command line, and then do `os.environ.get('USERNAME)` for example

Reply:
Just changed it, the correct syntax is os.environ['USERNAME'] after on your machine you do export USERNAME=(put username here)